### ISSUE TITLE
[tune] Track & visualize metrics of all trials using PyTorch?

### ISSUE THREAD

**Issue by nyxynyx** [ray-project/ray#8149]
## Is it possible to track and visualize the metrics (loss, accuracy, etc) of all trials (terminated & running) on a dashboard when using PyTorch?

Something similar to Tensorboard, but ideally as simple as possible, as I will just need to visualize the training metrics.

![image](https://user-images.githubusercontent.com/8941374/80121413-463a5400-855a-11ea-8b99-049d8b4dcaf3.png)

*Ray version and other system information (Python version, TensorFlow version, OS):*
- ray 0.8.4
- Python 3.7.3
- Pytorch 1.3.1
- Ubuntu 18.04

**Comment by richardliaw** [comment]
Does tensorboard not automatically show up for you?

if you do: `tune.run(..., )`, your results should show up if you do
```bash
tensorboard --logdir ~/ray_results
```

**Comment by nyxynyx** [comment]
Works after installing `tensorboard`, thanks!
